src/deepsparse/v2/text_generation/nl_engine_operator.py

```diff
@@ -59,17 +124,16 @@ def __init__(
         **kwargs,
     ):
 
+        self.sequence_length = sequence_length
+        self.input_ids_length = input_ids_length
         self.kv_cache_data_type = None
+        self.internal_kv_cache = internal_kv_cache
+        self.model_path = kwargs.get("model_path")
         (
             onnx_file_path,
             output_indices_to_be_cached,
             kv_cache_data_type,
-        ) = overwrite_onnx_model_inputs_for_kv_cache_models(
-            onnx_file_path=kwargs.get("model_path"),
-            batch_size=kwargs.get("batch_size", 1),
-            sequence_length=sequence_length,
-            input_ids_length=input_ids_length,
-        )
+        ) = self.override_model(self.model_path, batch_size=1)
```

Comment:
```suggestion
        ) = self.override_model_inputs(self.model_path, batch_size=1)
```

- The Verbose Damian